src/net.h

```diff
@@ -108,7 +108,7 @@ struct CSerializedNetMsg
     CSerializedNetMsg& operator=(const CSerializedNetMsg&) = delete;
 
     std::vector<unsigned char> data;
-    std::string command;
+    std::string msg_type;
```

Comment:
```suggestion
    std::string m_type;
```

According to the dev notes, members should be prefixed with `m_`.

Reply:
Thanks, done.